benchexec/tablegenerator/__init__.py

```diff
@@ -1382,10 +1386,201 @@ def write_head_line(
+                    "Skipping column %s for now" % (column_title, column_title)
+                )
+                continue
+
+            used_column_titles.add(column_title)
+            current_command.set_command_part("column_title", column_title)
+
+            column_statistic_to_latex_command(
+                current_command, column_stats, latex_commands, **column.__dict__
+            )
+
+    out.write(header)
+    out.write("\n".join(command.to_latex_raw() for command in latex_commands))
+
+
+class LatexCommand:
+    """Data holder for latex command."""
+
+    def __init__(self, bench_name="", runset_name=""):
+        self.bench_name = LatexCommand.format_command_part(str(bench_name))
+        self.runset_name = LatexCommand.format_command_part(str(runset_name))
+        self.column_title = ""
+        self.column_category = ""
+        self.column_subcategory = ""
+        self.stat_type = ""
+        self.value = None
+
+    def set_command_part(self, part_name: str, part_value) -> "LatexCommand":
+        """Sets the value of the command part
+
+        Available part names:
+            bench_name, runset_name, column_title, column_category, column_subcategory, stat_type
+
+        Args:
+            part_name: One of the names above
+            part_value: The value to be set for this command part
+
+        Returns:
+            This LatexCommand
+        """
+        self.__dict__[part_name] = LatexCommand.format_command_part(str(part_value))
+        return self
+
+    def set_command_value(self, value, scale_factor, **value_data) -> "LatexCommand":
+        """Sets the value for this command
+
+        Args:
+            value: The new command value
+            scale_factor: Scaling factor for the new value
+            value_data: Remaining unused value_data
+
+        Returns:
+            This LatexCommand
+        """
+        self.value = decimal.Decimal(value)
+        self.value *= scale_factor
+        return self
+
+    def to_latex_raw(self) -> str:
+        """Prints latex command with raw value (e.g. only number, no additional latex command)."""
+        return self.__get_command_formatted(util.print_decimal(self.value))
+
+    def __repr__(self):
+        return "\\StoreBenchExecResult{%s}{%s}{%s}{%s}{%s}{%s}" % (
+            self.bench_name,
+            self.runset_name,
+            self.column_title,
+            self.column_category,
+            self.column_subcategory,
+            self.stat_type,
+        )
+
+    def __get_command_formatted(self, value: str) -> str:
+        """Formats the command with all parts and appends the value
+
+        To use a custom format for the value, for example
+            \\StoreBenchExecResult{some}{stuff}...{last_name_part}{\\textbf{value}}
+        format the value and give it to this function
+        """
+        if not value:
+            logging.warning(
+                "Trying to print latex command without value! Using 0 as value for command:\n %s"
```

Comment:
We must not dump wrong values. If there is no value to print, then we cannot invent the number `0` (why `0` and not `42`?). So please make it print an empty value.